src/components/CardListItem.js

```diff
@@ -129,17 +129,34 @@ const renderCardContent = (item, index, horizontal, noOvertitle, bigTitle, sue)
     subtitle && cardContent.push(sequenceMap.subtitle());
 
     if (sue) {
-      !picture?.url && cardContent.push(sequenceMap.sue.pictureFallback());
-      serviceName && requestedDatetime && cardContent.push(sequenceMap.sue.category());
-      serviceName && requestedDatetime && cardContent.push(sequenceMap.sue.divider());
-      title &&
-        cardContent.push(
-          <WrapperHorizontal key={keyExtractor(title, index)}>
-            {sequenceMap.title()}
-          </WrapperHorizontal>
-        );
-      address && cardContent.push(sequenceMap.sue.address());
-      status && cardContent.push(sequenceMap.sue.status());
+      const addSueContent = () => {
+        if (!picture?.url) {
+          cardContent.push(sequenceMap.sue.pictureFallback());
+        }
+
+        if (serviceName && requestedDatetime) {
+          cardContent.push(sequenceMap.sue.category());
+          cardContent.push(sequenceMap.sue.divider());
+        }
+
+        if (title) {
+          cardContent.push(
+            <WrapperHorizontal key={keyExtractor(title, index)}>
+              {sequenceMap.title()}
+            </WrapperHorizontal>
+          );
+        }
+
+        if (address) {
+          cardContent.push(sequenceMap.sue.address());
+        }
+
+        if (status) {
+          cardContent.push(sequenceMap.sue.status());
+        }
+      };
+
+      addSueContent();
```

Comment:
why do you add an function here that checks conditions instead of directly calling the conditions? the function is also called every time.

what about

```
if (sue) {
  if (!picture?.url) {
    cardContent.push(sequenceMap.sue.pictureFallback());
  }

  if (serviceName && requestedDatetime) {
    cardContent.push(sequenceMap.sue.category());
    cardContent.push(sequenceMap.sue.divider());
  }

  if (title) {
    cardContent.push(
      <WrapperHorizontal key={keyExtractor(title, index)}>
        {sequenceMap.title()}
      </WrapperHorizontal>
    );
  }

  if (address) {
    cardContent.push(sequenceMap.sue.address());
  }

  if (status) {
    cardContent.push(sequenceMap.sue.status());
  }
};
```

Reply:
I added a log inside the if(sue) control. There are 2 items in the list. In the old version of the code I can see this log 4 times. But in the new version of the code I can see the log 2 times. So it is called 1 time for each list item. I think this is acceptable. 

Also, looking at the previous version of the code, it is more difficult for me to understand.

Reply:
yes, the previous version is nothing we want. i was speaking about the additional method you want to introduce for some reason.

Reply:
I thought about it again. I decided that we don't need an extra method. I just changed the && operator control into an if control. Logically it works in the same way as the && operator. But I think it is more readable this way.